s were configured automatically based

```diff
@@ -0,0 +1,39 @@
+diff.astextplain.textconv=astextplain
```

Comment:
I think this file should be deleted, @robvet

Reply:
Yes, thx!

Reply:
Not seeing that line of code anywhere in repo. Please advise.  Better yet, if it's there, please delete it.

Reply:
I just deleted the file.